docs/resources/project.md

```diff
@@ -129,6 +128,10 @@ lifecycle {
 ```
 - `role` (Block Set) Project role. Element has one to one mapping with the [JFrog Project Roles API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-AddaNewRole) (see [below for nested schema](#nestedblock--role))
 
+### Read-Only
+
+- `id` (String) The ID of this resource.
```

Comment:
Is it the change on the doc generator side?

Reply:
Yes, this is from the updated `terraform-plugins-docs` module.